npm.js

```diff
@@ -66,6 +73,8 @@ exports.translate = function(load){
 				if(pkg.browser){
 					delete pkg.browser.transform;
 				}
+				pkg = transform(loader, load, pkg);
```

Comment:
The jsonTransform in the system-json extension occurs when a user does:

```js
var json = require("foo.json");
```

In their code. Of course, for the npm plugin no such code exists. This is why it doesn't go through the json extension.  

What I'm saying is, the json files that we load in system-npm do not have load objects because *nothing imports them*.  We [fetch them here](https://github.com/stealjs/system-npm/blob/master/npm-crawl.js#L416) but fetching is just an ajax call, it doesn't fully import them through the loader.

Therefore, they do not have load objects, so if you want to support the jsonOptions.transform api, you'll have to make a fake load object to pass into that function.